src/pages/Onboarding/pages/Seedless/components/ExistingWalletButton.tsx

```diff
@@ -18,11 +18,12 @@ export function ExistingWalletButton({
         width: '225px',
         height: '180px',
         borderRadius: 2,
-        backgroundColor: 'grey.700',
+        backgroundColor: 'rgba(88, 88, 91, 0.75)',
```

Comment:
Using explicit values with `rgba` to make sure the text is not transparent, only the background